pkg/protocol/xprotocol/dubbo/mapping.go

```diff
@@ -0,0 +1,37 @@
+package dubbo
+
+import (
+	"context"
+	"errors"
+	"mosn.io/mosn/pkg/protocol/xprotocol"
+	"mosn.io/mosn/pkg/types"
+	"net/http"
+)
+
+func init() {
+	xprotocol.RegisterMapping(ProtocolName, &dubboStatusMapping{})
+}
+
+type dubboStatusMapping struct{}
+
+func (m *dubboStatusMapping) MappingHeaderStatusCode(ctx context.Context, headers types.HeaderMap) (int, error) {
+	cmd, ok := headers.(xprotocol.XRespFrame)
+	if !ok {
+		return 0, errors.New("no response status in headers")
+	}
+	code := uint16(cmd.GetStatusCode())
+	switch code {
+	case 20:
```

Comment:
You can use [status](https://github.com/mosn/mosn/blob/b2d8718b12ab1f57f09d85042c24b6bd09b8cd19/pkg/protocol/xprotocol/dubbo/types.go#L63) replace number

Reply:
done

Reply:
include unit test, use `pkg/types/constant.go` const